yarn.lock

```diff
@@ -8706,7 +8706,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"jest-environment-puppeteer@npm:^6.0.3":
+"jest-environment-puppeteer@npm:^6.0.2, jest-environment-puppeteer@npm:^6.0.3":
```

Comment:
What is requiring 6.0.2?

Reply:
I don't think anything is requiring ver 6.0.2. I think I messed it up during `yarn add`. I upgraded version in package.json just now. Ver 6.0.2 is gone.